string drink_name
---
int8[] Ingredient_list